### ISSUE TITLE
[FEATURE] Add support for setter method coexisting with fluent setter method.

### ISSUE THREAD

**Issue by jearton** [rzwitserloot/lombok#2141]
**Describe the feature**

```java
@Setter
@FluentSetter
public class User {
    private String id;
}
```

will generate：

```java
public class User {
    
    private String id;
    
    public void setId(String id) {
        this.id = id;
    }

    public User id(String id) {
        this.id = id;
        return this;
    }
}
```

**Describe the target audience**

The standard setter method is useful to  Json serialization tool like  `jackson`. And fluent setter method is developer friendly when coding.

**Comment by janmaterne** [comment]
What's wrong with the existsing:

```java
@Setter
@Accessors(chain = true)
public class User {

	private String id;
	private String name;

	public static void main(String[] args) {
		User user = new User()
			.setId("42")
			.setName("Douglas Adams");
	}
	
}
```

**Comment by jearton** [comment]
I want to keep the setter methods returning `void`. And then add extra fluent setter methods to return `this`. A framework of my company can only recognize the standard setter methods of returning void.  -_-!

**Comment by janmaterne** [comment]
You cant have both.

```
void setId(String id)
String setId(String id)
```

Both methods have the same signature and can't coexist.

Maybe @Wither is your way?
https://projectlombok.org/features/experimental/Wither

**Comment by Saljack** [comment]
No he wants
```java
void setId(String id);
User id(String id);
```
I don't want setters with returning "this" but I would like have both a setter `void setXXXX(Param param)` and a fluent method `ClassName XXX(Param param)`

```java
User user = new User().id("id");

user.setId("id");
```

**Comment by pnavato** [comment]
Yes, the best solution would be having traditional getters/setters along with fluent setters (without the "set" prefix).
Non-void setXXX methods **break the JavaBean specification** and are **not accepted by many frameworks** (see, for example, [@Accessors(fluent = true) does not work with Jakson](https://stackoverflow.com/questions/72570822/accessorsfluent-true-does-not-work-with-jakson) and [Spring ConfigurationProperties to work with fluent setters or custom setters](https://stackoverflow.com/questions/57308752/spring-configurationproperties-to-work-with-fluent-setters-or-custom-setters)) so they should be simply discarded.
This feature has been requested multiple times, both here (https://github.com/projectlombok/lombok/issues/1595 and https://github.com/projectlombok/lombok/issues/3265) and on [Stack Overflow](https://stackoverflow.com/q/36564153/685806) but yet nothing happened.

**Comment by janrieke** [comment]
BTW: I'd like to know which (public) frameworks are affected by this kind of problem. E.g., if it's Jackson, we can talk about an extension to `@Jacksonized` to automatically configure Jackson such that fluent and/or chainable work.

**Comment by rzwitserloot** [comment]
We don't like this. JanRieke correctly explained why.

**Comment by janrieke** [comment]
BTW: Jackson indeed has problems with fluent accessors (chained is fine though).
I think we can upgrade `@Jacksonized` to mitigate this. I'll investigating this in the next weeks (can't promise too much here, I'm rather busy ATM).

**Comment by adnsimona** [comment]
Hi, I got across this topic because we would like to use fluent getters but javabeans setters with Lombok.
So a `@FluentGetter` would be also nice to have next to `@FluentSetter`.
